src/services/StoreService.ts

```diff
@@ -0,0 +1,17 @@
+import AsyncStorage from '@react-native-community/async-storage';
+
+export default class StoreService {
+  async getStoreValue(key: string): Promise<string | null> {
+    try {
+      const value = await AsyncStorage.getItem(key);
+      return value;
+    } catch (_) {
+      return '';
+    }
+  }
+
+  async setStoreValue(key: string, value: string | boolean | number): Promise<void> {
+    await AsyncStorage.setItem(key, JSON.stringify(value));
+    return;
+  }
+}
```

Comment:
is it necessary if we have it in redux?

Reply:
yes redux keep sync only on some scenario